exporters/elasticsearch/include/opentelemetry/exporters/elasticsearch/es_log_recordable.h

```diff
@@ -31,13 +31,8 @@ class ElasticSearchRecordable final : public sdk::logs::Recordable
    * A helper method that writes a key/value pair under a specified name, the two names used here
    * being "attributes" and "resources"
    */
-  void WriteKeyValue(nostd::string_view key,
-                     const opentelemetry::common::AttributeValue &value,
-                     const std::string &name);
-
-  void WriteKeyValue(nostd::string_view key,
-                     const opentelemetry::sdk::common::OwnedAttributeValue &value,
-                     const std::string &name);
+  void WriteValue(const opentelemetry::sdk::common::OwnedAttributeValue &value,
```

Comment:
nit - do we need to revisit the method comment above?

Reply:
Yes probably
This is not needing anymore so I guess we just either right a small comment or just no comment at all since the `WriteValue` methods are kind of self explanatory ?

Reply:
I did some changes in the comment. Let me now if this is good for you.